build.gradle

```diff
@@ -108,7 +108,10 @@ docker {
     name "${REPOSITORY_URI}"
     tags "${TAG}", 'latest'
     files jar.archivePath
-    buildArgs(['JAR_FILE': "${bootJar.archiveName}"])
+    buildArgs([
+        'JAR_FILE': "${bootJar.archiveName}",
+        'APPLICATION_CONTEXT': "sbjb/"
```

Comment:
Do you want to put a note about updating this in the readme? Could be forgotten otherwise.

Reply:
Cool - will do.